Makefile

```diff
@@ -32,3 +34,18 @@ release: b2d_iso get_gh-release
 	cp $(BUILD_DIR)/minishift-b2d.iso release/
 	$(BUILD_DIR)/gh-release checksums sha256
 	$(BUILD_DIR)/gh-release create minishift/minishift-b2d-iso $(VERSION) master v$(VERSION)
+
+.PHONY: prepare_for_test
+prepare_for_test: DOCKER_MACHINE_PATH=$(BIN_DIR)/docker-machine
```

Comment:
Also, the Makefile seems to hardcode a Linux install. It would be nice if the right minishift version would be downloaded, so that for example these smoke tests can be run on OS X as well. I would not make this a requirement for getting this merged, but in the very least we would need a follow up issue trying to address OS problems.